src/core/Clock.js

```diff
@@ -167,3 +177,18 @@ getJasmineRequireObj().Clock = function() {
 
   return Clock;
 };
+
+/**
+ * Stubs Node.js Timeout class
+ */
+function Timeout(id) {
+  this.id = id;
```

Comment:
I'd rather not give these fake `Timeout` objects an `id` property that the real ones don't have. It looks like all we're doing with it is passing it back to `delayedFunctionScheduler.removeFunctionWithId`. A better approach might be to use the `Timeout` object itself as a timeout key, using the optional 5th argument to `delayedFunctionScheduler.scheduleFunction`. Does that make sense?